fe2-android/app/src/main/java/com/github/dedis/popstellar/model/network/method/message/data/digitalcash/ScriptInput.java

```diff
@@ -0,0 +1,75 @@
+package com.github.dedis.popstellar.model.network.method.message.data.digitalcash;
+
+import com.google.gson.annotations.SerializedName;
+
+import java.util.Objects;
+
+// The script describing the unlock mechanism
+public class ScriptInput {
+  @SerializedName("type")
+  private final String type; // The script describing the unlock mechanism
+
+  @SerializedName("pubkey")
+  private final String pubKeyRecipient; // The recipient’s public key
```

Comment:
Would it be possible to ues the PublicKey type here ?
It would make it much more readable.

(The is already a rule to parse `base64 strings` -> `Public Key` for the json serialization)